docker/maistra-builder_2.5.Dockerfile

```diff
@@ -0,0 +1,261 @@
+FROM quay.io/centos/centos:stream8
```

Comment:
You mentioned something about openssl 3.0. Does this base image contain what you need?

Reply:
I think it does not, but we will not need OpenSSL 3.0 until proxy 2.5 is ready to use, so I will do it later.